### DISCUSSION TITLE
Is it possible to include the test source code with each test result?

### DISCUSSION THREAD

**Discussion by ajwood** [pytest-dev/pytest-html#475] [Q&A]
The default pytest output for failed tests includes the whole function definition for the failed test, which ends up embedded in the html report. Is there a way to get the function definitions for _passed_ tests to be embedded in the html report also? The idea is that someone reviewing a report might be interested in knowing exactly what was tested in any given scenario, and may not be able to easy access the actual test source code.

**Comment by BeyondEvil** [comment]
Yeah, we can probably have a query parameter to enable that.

**Comment by BeyondEvil** [comment]
Time flies.

I don't think this is possible as we're dependent on what gets passed to us by pytest (core). Afaict we only get the test source on failures.
